README.md

```diff
@@ -71,51 +71,58 @@ For example, to use `ruff-lsp` with Neovim, install `ruff-lsp` from PyPI along w
 to your `init.lua`:
```

Comment:
We need to update this paragraph since we will be omitting the code snippet. A sentence guiding the user to use the `nvim-lspconfig` documentation for setting up the LSP client will replace the last sentence here.